.github/workflows/test.yml

```diff
@@ -20,10 +20,12 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ['3.9', '3.10', '3.11', '3.12']
+        python-version: ['3.10', '3.11', '3.12']
```

Comment:
dropped 3.9 due to: https://github.com/wemake-services/dump-env/pull/387#discussion_r2089851496